tests/backend_test/main.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+#. ../assert.sh continue_on_error #  load test functions with continue_on_error/stop_on_error
+
+#Cleanup
+terraform init -backend-config=./test_backend.tfvars test3
+terraform destroy -auto-approve test3
+
+#Check if resource group exists
+#Resource group does not exists
+assertRG core-usea-rg-test not
```

Comment:
We're essentially absorbing any assert failures with this approach.  For example, I create this resource group and then ran the test.  The output on the screen showed the test failed, but it then just continued successfully with the rest of the test.  In other words, $? reported 0 even though one of the tests failed.  If any test fails, the script should exit then with an exit code of 1.

Reply:
This is mainly because main.sh is currently executing as `assert.sh continue_on_error` and not `assert.sh stop_on_error`. Thats part of the readme under parent test folder from where the execution for backend tests is called from.